src/AWS.Deploy.Orchestration/Utilities/DeployedApplicationQueryer.cs

```diff
@@ -76,6 +89,19 @@ public async Task<List<CloudApplication>> GetExistingDeployedApplications(IList<
                 apps.Add(new CloudApplication(stack.StackName, recipeId));
             }
 
+            if (_deploymentManifestEngine != null &&
+                _session != null)
+            {
+                var deploymentManifest = await _deploymentManifestEngine.GetDeploymentManifest();
+                var lastDeployedStack = deploymentManifest?.LastDeployedStacks
+                    .FirstOrDefault(x => x.AWSAccountId.Equals(_session.AWSAccountId) && x.AWSRegion.Equals(_session.AWSRegion));
+
+                return apps.OrderBy(x => {
```

Comment:
I believe the plan was to sort the list of the stacks whether they are in or not of the `LastDeployedStacks` and then the stacks last update date. That way the stack you most recently deployed to when show up first. We should also return the last updated date in case the toolkit wants to show it so the users can see the order is date based and not alpha.

Reply:
Fixed